deployments/cloudfoundry/bosh/DEVELOPMENT.md

```diff
@@ -6,19 +6,40 @@
 
 [Quick Start Guide](https://bosh.io/docs/bosh-lite/)
 
-### Supplementary Getting Started Checklist
+### Start the Bosh Director
 
-- Make sure director is running
-    - Quick start guide to run a BOSH Director can be found [here.](https://bosh.io/docs/quick-start/)
-- Need to upload right Ubuntu/OS blob
-    - Guide can be found [here.](https://bosh.io/docs/uploading-stemcells/)
-    - The following is an example command to upload the Warden (BOSH Lite) Ubuntu Bionic (18.04.6 LTS) stemcell:
+Quick start guide to run a BOSH Director can be found [here.](https://bosh.io/docs/quick-start/)
```

Comment:
The quickstart guide is pretty minimal so it may be a good idea to include what you used at the time of these updates as a record. Otherwise going into the debug instructions assumes a lot of context that is hard to follow.

Reply:
I moved some stuff around to hopefully make it more clear. Let me know if it makes better sense now. The idea is just start the Bosh director and upload the ubuntu blob, so hopefully the new setup makes that clear.